### PR TITLE
remove containsKey() from validateParams()

### PR DESCRIPTION
**Pull request by KevinMenden**
If we use `.keySet().contains()`  instead of `.containsKey()`, the problems at https://github.com/nf-core/diaproteomics/pull/103/checks?check_run_id=2167429157 are fixed.

### CODE REVIEW

**Comment by KevinMenden** [comment]
closes https://github.com/nf-core/tools/issues/937

File: nf_core/lint/nextflow_config.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def nextflow_config(self):
         "params.container",
         "params.singleEnd",
         "params.igenomesIgnore",
+        "params.name",
```
Comment (ewels) [inline nf_core/lint/nextflow_config.py:119]: 👍🏻

File: nf_core/pipeline-template/lib/NfcoreSchema.groovy [hunk 2]
```diff
@@ -219,7 +219,7 @@ class NfcoreSchema {
                     }
                 }
             }
-            if(rawSchema.keySet().contains('properties') && rawSchema.get('properties').containsKey(ignore_param)) {
+            if(rawSchema.keySet().contains('properties') && rawSchema.get('properties').keySet().contains(ignore_param)) {
```
Comment (ewels) [inline nf_core/pipeline-template/lib/NfcoreSchema.groovy:222]: ah right, yeah if nothing else this looks more consistent too 👍🏻 

I guess that we changed the initial call at some point but as we never tested on a pipeline that had ungrouped params we didn't see the failure / didn't update the second call..

**Review by ewels — APPROVED** [review]
Nice!

File: nf_core/pipeline-template/lib/NfcoreSchema.groovy (hunk 2 above)
Comment (KevinMenden) [reply nf_core/pipeline-template/lib/NfcoreSchema.groovy:222]: Yep, I had a feeling this could happen with exactly this part of the code :/

**Pull request merged**